### PR TITLE
Add Alibaba Cloud EventBridge integration

### PR DESCRIPTION
**Pull request by wanghq**
Also add the npm install step to readme

### CODE REVIEW

**Comment by wanghq** [comment]
@duglin , please let me know if I need two tweak anything. Thanks!

File: data/integrations.yaml [hunk 1]
```diff
@@ -43,3 +43,8 @@
     Debezium, a distributed open-source change data capture platform, can emit change data events [in the CloudEvents format](https://debezium.io/documentation/reference/integrations/cloudevents.html)
   url: https://debezium.io/
   logo: debezium.png
+- name: Alibaba Cloud EventBridge
+  description: |
+    A serverless event bus that receives CloudEvents compliant events from cloud services, SaaS/custom apps, and routes to various targets.
```
Comment (duglin) [inline data/integrations.yaml:48]: Minor thing, but can you remove the trailing period to match the others?

Comment (wanghq) [reply data/integrations.yaml]: Removed, thanks!

**Comment by duglin** [comment]
thanks!

**Pull request merged**